astor/rtrip.py

```diff
@@ -85,6 +85,10 @@
 dsttree = 'tmp_rtrip'
 
 
+def out_prep(s, pre_encoded=(type(b'') is type(''))):
```

Comment:
I agree that there is no need for an overkill solution. Something like `sys.version_info[0] == 2` would probably work.